File: bastion/util/TabBastion.scala
package bastion.util

import net.minecraft.item.ItemStack
import net.minecraft.creativetab.CreativeTabs

class TabBastion extends CreativeTabs("ChaoticBastion") {
    var display: ItemStack

    def init(stack: ItemStack) {
        display = stack
    }

    @Override
    def getIconItemStack() = display
}

File: src/bastion/util/TabBastion.java
package bastion.util;

import net.minecraft.creativetab.CreativeTabs;
import net.minecraft.item.ItemStack;

public class TabBastion extends CreativeTabs
{
    ItemStack display;

    public TabBastion(String name)
    {
        super(name);
    }

    public TabBastion init (ItemStack stack)
    {
        display = stack;
        return this;
    }

    public ItemStack getIconItemStack ()
    {
        return display;
    }
}